src/main/java/forceitembattle/settings/preset/GamePreset.java

```diff
@@ -30,39 +32,20 @@ public String presetName() {
         return presetName;
```

Comment:
At this point we could just switch to Kotlin xD

Reply:
yea I dont know why but I like the use of record-style methods xd but I changed it and added @Getter lombok there in the next commit

Reply:
ah no worries you can use it xD
I kinda like it too ngl.
Was just meant as a comment. Kotlin looks very close to this.